backend-test/startGame/startGame.js

```diff
@@ -0,0 +1,59 @@
+import 'babel-polyfill';
+import request from 'supertest';
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import app from '../../server';
+
+const User = mongoose.model('User');
+
+const mockPlayers = {
+  gameStarter: 'Kevin',
+  gameWinner: 'kevin',
+  players: [1, 2, 3]
+};
+const userMock = {
+  name: 'kelvin',
+  password: '12345',
+  username: 'kelvin',
+  email: 'kelvin@email.com'
+};
+let token = '';
+
+describe('Player endpoints', () => {
+  before(() => {
+    Promise.resolve(User.create(userMock));
+  });
+  it('POST /api/auth/endpoint should return the user token along with the', (done) => {
```

Comment:
The comment states that `POST /api/auth/endpoint should return the user token along with the`, along with the what?